app/api/settings/route.ts

```
@@ -12,27 +12,52 @@ export async function PUT(req: NextRequest) {
-    const { enableEmailCapture, workspaceId: bodyWorkspaceId } = body;
+    const { enableEmailCapture, workspaceId: bodyWorkspaceId, webhookUrl } = body;
 
     const preferredWorkspaceId = req.headers.get("x-workspace-id") || req.nextUrl?.searchParams?.get("workspaceId") || bodyWorkspaceId;
     const workspace = await resolveActiveWorkspace(session.user.id, preferredWorkspaceId);
     if (!workspace) {
       return NextResponse.json({ error: "Workspace not found" }, { status: 404 });
     }
 
-    if (typeof enableEmailCapture !== "boolean") {
-      return NextResponse.json({ error: "enableEmailCapture must be a boolean" }, { status: 400 });
+    let updateData: any = {};
```

ESLint

[error] 'updateData' is never reassigned. Use 'const' instead.

(prefer-const)